### PR TITLE
Result Pie Charts

### PR DESCRIPTION
**Pull request by dasGoogle**
Displays pie charts for the voting results. 
Merge #14 first, since this branch relies on these changes. 

image

### CODE REVIEW

File: frontend/src/components/QuestionResults.tsx [hunk 1]
```diff
@@ -1,16 +1,60 @@
...
 export default function QuestionResults(props: QuestionResultsProps) {
+  const series = [];
+  const labels = [];
+
+  // The order of these changes on database changes by voting, to maintain a stable chart, we need to sort the answers by name
+  const answers = props.question.answers.sort(answerSorter);
+
+  for (const answer of answers) {
+    series.push(answer.count);
+    labels.push(answer.text);
```
Comment (rgwohlbold) [inline frontend/src/components/QuestionResults.tsx:19]: It would be nicer to use `const series = answers.map((answer) => answer.count)` after sorting

**Review by rgwohlbold — APPROVED** [review]
Looks good. This will not be as nice for longer answers (some lectures do have these). The colors are also not the easiest to discern, that could be changed at some point as well. The priority should be to get this merged before Wednesday so we can give a small Live-Demo :)

File: frontend/src/components/QuestionResults.tsx (hunk 1 above)
Comment (dasGoogle) [reply frontend/src/components/QuestionResults.tsx:19]: You certainly have a point here, I will fix that.

**Comment by dasGoogle** [comment]
> Looks good. This will not be as nice for longer answers (some lectures do have these). The colors are also not the easiest to discern, that could be changed at some point as well. The priority should be to get this merged before Wednesday so we can give a small Live-Demo :)

I have created issues #16 and #17 for this, when we have a look at this again.

**Comment by Greenscreen23** [comment]
I'd like to review this one after #14 is merged, as it is easier to see the diff, or is there some way to this in github that I'm missing

**Comment by dasGoogle** [comment]
> I'd like to review this one after #14 is merged, as it is easier to see the diff, or is there some way to this in github that I'm missing



> I'd like to review this one after #14 is merged, as it is easier to see the diff, or is there some way to this in github that I'm missing

I dont think so, however all changes made by me are in the following file: https://github.com/solid-polls/solid/blob/ba0fbbc57765b74685ab50e2da0be35d9742c65a/frontend/src/components/QuestionResults.tsx

**Review by Greenscreen23 — APPROVED** [review]
If that's your only change, then I guess this looks fine :)

**Review by lukasrad02 — APPROVED** [review]

**Pull request merged**